bc/core/utils/messages.py

```diff
@@ -68,6 +68,7 @@ def _available_space(self, *args, **kwargs) -> int:
     def format(self, *args, **kwargs) -> str:
         if "description" in kwargs:
             available_space = self._available_space("description", **kwargs)
+            print(available_space)
```

Comment:
Oops.

Reply:
Can you fix with an amended commit and a force-push so our commit log isn't too polluted?

Reply:
Sure!